.ruby-version

```diff
@@ -0,0 +1 @@
+2.7.2
```

Comment:
nice :sunglasses:

Reply:
That might have been accidental; I'd have done it in a separate commit if I'd have noticed.  But yeah, at least it's done.